examples/basic_tutorial_BERT.ipynb

```diff
@@ -0,0 +1,1198 @@
+{
```

Comment:
<p>can you please add a few inline comments to describe what you are doing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-deep/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>